src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_help.py

```diff
@@ -88,11 +99,18 @@
     type: group
     short-summary: Commands to manage deployment operations
 """
-helps['resource provider'] = """
+helps['provider'] = """
     type: group
-    short-summary: Commands to manage resource providers
+    short-summary: Manage resource providers
+"""
+helps['provider register'] = """
```

Comment:
I don't see the value of moving this to the top when it has four commands and no subgroups.